src/user/system/LoadBalancerManager.ts

```diff
@@ -628,7 +628,15 @@ class LoadBalancerManager {
                             protocol: 'udp',
                             publishMode: 'host',
                             containerPort: 443,
-                            hostPort: 443,
+                            hostPort:
+                                CaptainConstants.configs.nginxPortNumber443,
+                        },
+                        {
+                            protocol: 'udp',
+                            publishMode: 'host',
+                            containerPort: 80,
```

Comment:
Why do we need udp/80?

Reply:
We don't removed. We will need for http/3 as was needed for 443 port. But is not important, better to be done in other commit.